### DISCUSSION TITLE
Waterfall Type Active Notes Persist

### DISCUSSION THREAD

**Discussion by zhuzixu900818** [cifkao/html-midi-player#38] [Ideas]
Hi,
In the Waterfall visualizer, the active notes got highlight only at the onset of the note, is there anyway I can set the an note to be active as long as there is a "pillar" right on top it, meaning the note highlight stays while it's being pressed?
Thank you!

**Comment by cifkao** [comment]
Hi,
I know the active note highlighting is not ideal, but this is an issue with [`@magenta/music`](https://github.com/magenta/magenta-js), so unfortunately not something I can fix here.
